sites/authentication/authentication-en.html

```diff
@@ -0,0 +1,89 @@
+---
+{
+	"altLangPage": "authentication-fr.html",
+	"breadcrumb": {
+		"title": "Canada.ca", "link": "https://www.canada.ca/en.html"
+	},
+	"dateModified": "2022-05-04",
+	"description": "Documentation on how to use the various authentication patterns.",
+	"language": "en",
+	"title": "Authentication patterns"
+}
+---
```

Comment:
Just to add a little bit more context about the authentication component and it's variant
```suggestion
---
<p>This component support the following 3 design pattern variant</p>
<ul>
   <li>Contextual Sign-in button: A link pointing to a sign-in dedicated web page</li>
   <li>Signed-off: A pattern that show the user has been logged in and how it can deconnect</li>
   <li>Signed-on: A pattern that show how the user can sign-on directly on the current page. Note: The prompt dialog and the interaction resulting of the signed-on are out-of-scope currently.</li>
</ul>
```